docs/quickstart.md

```diff
@@ -0,0 +1,177 @@
+# Quickstart with firecracker-containerd
+
+This quickstart guide provides simple steps to get a working
+firecracker-containerd environment, with each of the major components built from
+source.  Once you have completed this quickstart, you should be able to run and
+develop firecracker-containerd (the components in this repository), the
+Firecracker VMM, and containerd.
+
+This quickstart will clone repositories under your `$HOME` directory and install
+files into `/usr/local/bin`.
+
+1. Get an AWS account
```

Comment:
Is it worth linking to https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/?